src/govuk/components/tag/_tag.scss

```diff
@@ -41,4 +43,50 @@
   .govuk-tag--inactive {
     background-color: govuk-colour("dark-grey", $legacy: "grey-1");
   }
+
+  .govuk-tag--grey {
```

Comment:
We should consider whether there's a need for `inactive` and `grey`, or whether they can be the same thing.

Reply:
Yeah. I considered deleting it, but I thought that would delay the contribution as I assumed it would be a breaking change.

Reply:
We could consider marking it as deprecated, for example.

Reply:
Yeah, and actually, it's not documented on the Tag component page. Is it documented publically elsewhere?

Reply:
I've left it in, and maybe the deprecation can be a release note? Or do I need to do something in the code to mark it as deprecated?

Reply:
Since it's not been documented we could consider dropping it in 4.0.0

Reply:
Created an issue for dropping it in 4.0.0: https://github.com/alphagov/govuk-frontend/issues/1757  

I'll chat to @m-green about whether we want to add a deprecation notice for `govuk-tag--inactive` somewhere, seeing that we don't document the class anywhere currently.

Reply:
@adamsilver Would you be okay to add something like
```
Deprecated. We will be removing this class in a future release, use `.govuk-tag--grey` instead.
```
above the `govuk-tag--inactive` class in the scss file? 🙏

Reply:
No probs - added in https://github.com/alphagov/govuk-frontend/pull/1756/commits/ed8534b156ad53c32dd71240b81a12b243f747a5

Reply:
@hannalaakso where would you like me to put this line?

Reply:
@hannalaakso done.